src/Objs/Product/ProductComponent.tsx

```diff
@@ -33,6 +34,7 @@ provideComponent(Product, ({ page }) => {
                   content={page}
                   attribute="image"
                   className="img-background"
+                  alt={alternativeTextForObj(page.get('image'))}
```

Comment:
* [ ] Looking at product pages and the alt decision tree I'd say it doesn't contribute meaning/is redundant and thus should have an empty `alt`.